docs/manual/src/futures.md

```diff
@@ -0,0 +1,135 @@
+# Async/Future support
+
+UniFFI supports exposing async rust functions over the FFI. It can convert a Rust `Future<>/async fn` to and from foreign native futures (`async/await` in Python/Swift, `suspend fun` in Kotlin etc.)
```

Comment:
```suggestion
UniFFI supports exposing async rust functions over the FFI. It can convert a Rust `Future`/`async fn` to and from foreign native futures (`async`/`await` in Python/Swift, `suspend fun` in Kotlin etc.)
```